### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by annetill**
**Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

No.

**What kind of change does this PR introduce?**

It is an update of `README` file with the new features and the roadmap.

**What is the current behavior?**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [ ] No

**If yes, please check if the following requirements are fulfilled**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration steps are described in the following section

**What changes might users need to make in their application due to this PR? (migration steps)**

**Other information**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ PowSyBl Open Load Flow provides:
 - An open source implementation of the [SecurityAnalysis API from PowSyBl Core](https://www.powsybl.org/pages/documentation/simulation/securityanalysis/), we support either DC or AC calculations.
 - An open source implementation of the [SensitivityAnalysis API from PowSyBl Core](https://www.powsybl.org/pages/documentation/simulation/sensitivity/), we support either DC or AC calculations.
 
-Almost all of the code is written in Java. It only relies on native code for the [KLU](http://faculty.cse.tamu.edu/davis/suitesparse.html) sparse linear solver. Linux, Windows and MacOS are supported.
+Most of the code is written in Java. It only relies on native code for the [KLU](http://faculty.cse.tamu.edu/davis/suitesparse.html) sparse linear solver. Linux, Windows and MacOS are supported.
```
Comment (rosiereflo) [inline README.md:32]: Maybe you should specify the license type of KLU here?

File: README.md [hunk 2]
```diff
@@ -38,33 +38,39 @@ The AC calculations are based on full Newton-Raphson algorithm. The DC calculati
...
  - a support of generators' active and reactive power limits, included the support of reactive capability curves.
  - 5 starting point modes: flat, warm, only voltage angles initialization based on a DC load flow, only voltages magnitude initialization based on a specific initializer, or both voltages angle and magnitude initialization based on the two previous methods.
- - a support of non impedant branches, including complex non impedant sub-networks.
+ - a support of zero impedance branches, including complex zero impedance subnetworks, particularly important in case of voltage controls and topology changes involved in contingencies or in remedial actions.
  - a multiple synchronous component calculation, generally linked to HVDC lines.
+ - A modeling of secondary voltage control following research of [Balthazar Donon, Liège University](https://www.montefiore.uliege.be/cms/c_3482915/en/montefiore-directory?uid=u239564).
+ - Support of asymmetrical calculations.
```
Comment (Hadrien-Godard) [inline README.md:45]: Maybe a word on multiple state vector update methods in the Newton Raphson algorithm?
Something like: "- An implementation of three methods to update the state vector in the Newton Raphson algorithm: classic, rescaling under maximum voltage change and linear search rescaling."

File: README.md [hunk 3]
```diff
@@ -38,33 +38,39 @@ The AC calculations are based on full Newton-Raphson algorithm. The DC calculati
  - a distributed slack (on generation or on loads or on conform loads); Slack bus selection could be automatic or explicit as explained [here](https://www.powsybl.org/pages/documentation/simulation/powerflow/openlf.html#parameters).
  - a support of generators' active and reactive power limits, included the support of reactive capability curves.
  - 5 starting point modes: flat, warm, only voltage angles initialization based on a DC load flow, only voltages magnitude initialization based on a specific initializer, or both voltages angle and magnitude initialization based on the two previous methods.
- - a support of non impedant branches, including complex non impedant sub-networks.
+ - a support of zero impedance branches, including complex zero impedance subnetworks, particularly important in case of voltage controls and topology changes involved in contingencies or in remedial actions.
```
Comment (Hadrien-Godard) [inline README.md:41]: Maybe we should align the beginning of each sentence with a capital letter.

File: README.md [hunk 4]
```diff
@@ -38,33 +38,39 @@ The AC calculations are based on full Newton-Raphson algorithm. The DC calculati
...
- - a local and remote voltage control by transformers. It also supports shared controls between them. In case of a controlled bus that has both a voltage control by a generator and a transformer, we have decided in a first approach to discard the transformer control.
- - a local and remote voltage control by shunts. We also support shared controls between them. In case of a controlled bus that has both a voltage control by a generator and a shunt, we have decided in a first approach to discard the shunt voltage control. In case of a controlled bus that has both a voltage control by a transformer and a shunt, we have decided in a first approach to discard the shunt. Several shunts on a controller bus are supported. 
- - a remote reactive power control of a branch by a single generator connected on a bus.
+ - a local and remote voltage control by transformers. It also supports shared controls between them.
+ - a local and remote voltage control by shunts. We also support shared controls between them. Several shunts on a controller bus is supported.
+ - a remote reactive power control of a branch by generators. It also supports all type of shared controls between them.
+
```
Comment (Hadrien-Godard) [inline README.md:55]: Add: "- A remote reactive power control of a branch by transformers." ?

File: README.md (hunk 1 above)
Comment (annetill) [reply README.md:32]: @geofjamg I think it is https://github.com/DrTimothyAldenDavis/SuiteSparse/blob/dev/KLU/Doc/License.txt, if we only use this part, right?

File: README.md [hunk 5]
```diff
@@ -29,7 +29,7 @@ PowSyBl Open Load Flow provides:
 - An open-source implementation of the [SecurityAnalysis API from PowSyBl Core](https://www.powsybl.org/pages/documentation/simulation/securityanalysis/), supporting DC and AC calculations.
 - An open-source implementation of the [SensitivityAnalysis API from PowSyBl Core](https://www.powsybl.org/pages/documentation/simulation/sensitivity/), supporting DC and AC calculations.
 
-Most of the code is written in Java. It only relies on native code for the [KLU](http://faculty.cse.tamu.edu/davis/suitesparse.html) sparse linear solver. Linux, Windows and MacOS are supported.
+Most of the code is written in Java. It only relies on native code for the [KLU](http://faculty.cse.tamu.edu/davis/suitesparse.html) sparse linear solver. Linux, Windows and MacOS are supported. It is distributed with license LGPL-2.1+.
```
Comment (jeandemanged) [inline README.md:32]: KLU is distributed with license LGPL-2.1+.

**Review by jeandemanged — APPROVED** [review]

**Review by rosiereflo — APPROVED** [review]

**Pull request merged**